.dockerignore

```diff
@@ -1,2 +1,3 @@
 .git
 env_file
+.env
```

Comment:
Adding the env file that the docker compose stack uses to setup the environment for these scripts. It was already in the `.gitignore` file.